package/upgrade/upgrade_node.sh

```diff
@@ -17,8 +17,14 @@ clean_up_tmp_files()
     umount $target_elemental_cli || echo "Umount $target_elemental_cli failed with return code: $?"
   fi
   echo "Clean up tmp files..."
-  [[ -n "$NEW_OS_SQUASHFS_IMAGE_FILE" ]] && \rm -vf "$NEW_OS_SQUASHFS_IMAGE_FILE"
-  [[ -n "$tmp_rootfs_squashfs" ]] && \rm -vf "$tmp_rootfs_squashfs"
+  if [ -n "$NEW_OS_SQUASHFS_IMAGE_FILE" ]; then
+    echo "Try to remove $NEW_OS_SQUASHFS_IMAGE_FILE..."
+    \rm -vf "$NEW_OS_SQUASHFS_IMAGE_FILE"
```

Comment:
prefer to remove `\` if not required?

Reply:
The leading backslash is intended to bypass any possible aliases for `rm`. If this is not desirable or needed, I can remove it, thanks.

Reply:
Updated, thanks!